src/features/swap/SwapConfirm.tsx

```diff
@@ -111,7 +111,10 @@ export function SwapConfirmCard({ formValues }: Props) {
       return
     }
     const rateBN = new BigNumber(rate)
-    if (rateBN.lt(MIN_EXCHANGE_RATE) || rateBN.gt(MAX_EXCHANGE_RATE)) {
+    if (
+      ((toTokenId || fromTokenId) !== 'eXOF' && rateBN.lt(MIN_EXCHANGE_RATE)) ||
```

Comment:
Temporary fix to allow testing while we decide if we should keep this rate range check